src/api/java/mekanism/api/providers/IBlockProvider.java

```diff
@@ -22,12 +23,14 @@ default boolean blockMatches(Block other) {
     }
 
     @Override
+    @Nonnull
```

Comment:
Personally I am more of a fan of putting the Nonnull above the override as it is a shorter annotation